osu.Game/Overlays/Options/OptionsSubsection.cs

```diff
@@ -28,7 +29,7 @@ public OptionsSubsection()
                     Margin = new MarginPadding { Bottom = 10 },
                     Font = @"Exo2.0-Black",
                 },
-                Content = new FillFlowContainer

```

Comment:
EDIT: the difference between this one and all the others is that here it's an override